docs/src/tutorial-kinds-of-failure.md

```diff
@@ -231,30 +218,3 @@ Since this implementation is just the original one, but cast to a wider unsigned
 When Kani tells us both of these methods yield the same exact result, that gives us additional confidence that we haven't overlooked something.
 
 </details>
-
```

Comment:
We could replace this with a link to the limitation pages. What do you think?

Reply:
Good idea, I've restored in this PR the "Limitations" chapter (was removed in #969) and I'll add content to it after this is merged. I prefer that to going back to this file later to add the link.